clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala

```diff
@@ -59,4 +59,18 @@ trait ClioParsers {
   implicit val pathParser: ArgParser[Path] = parseWith("path", Paths.get(_))
 
   implicit val uuidParser: ArgParser[UUID] = parseWith("uuid", UUID.fromString)
+
+  implicit val stringParser: ArgParser[String] =
+    SimpleArgParser.from[String]("string") { str =>
+      if (str contains "\"") {
+        Left(
+          Error.MalformedValue(
+            "string",
+            "Quotes are not allowed in inputs"
+          )
+        )
+      } else {
+        Right(str)
+      }
```

Comment:
The `cats` library already in use in Clio provides a nice syntax for creating `Either`s based on a condition. This block can be written as:
```scala
Either.cond(
        !str.contains("\""),
        str,
        Error.MalformedValue("string", "Quotes are not allowed in inputs")
      )
```